graphkb/match.py

```diff
@@ -421,15 +421,15 @@ def match_positional_variant(
             )
     else:
         gene1 = parsed['reference1']
-    features = convert_to_rid_list(
-        get_equivalent_features(
-            conn,
-            gene1,
-            source=gene_source,
-            is_source_id=gene_is_source_id,
-            ignore_cache=ignore_cache,
-        )
+
+    gene1_features = get_equivalent_features(
+        conn,
+        gene1,
+        source=gene_source,
+        is_source_id=gene_is_source_id,
+        ignore_cache=ignore_cache,
     )
+    features = convert_to_rid_list(gene1_features)
```

Comment:
Just out of curiosity, is nested function calls a bad thing we should avoid or you are planning to need this **gene1_features** variable later on?

Reply:
Nested functions are fine.
I was just comparing gene1_features with gene2_features (below) while debugging, and thought it made the difference between features and secondary_features more clear.